docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,40 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component to share files among container components and collaborate with other teams during the development process.
```

Comment:
Later, we refer to `volume` components using monospace styling. Shouldn't we do the same here?